pymeasure/instruments/hp/hp8753e.py

```diff
@@ -0,0 +1,476 @@
+
+log = logging.getLogger(__name__)
+log.addHandler(logging.NullHandler())
+
+
+class HP8753E(Instrument):
+    """Represents the HP8753E Vector Network Analyzer
+    and provides a high-level interface for taking sweeps of the
+    scattering or measurement parameters.
+
+    :param name: Optional set the name of the instrument (str).
+    :param min_frequency: Optional set the minimum validator for `HP8753E.start_frequency`,
+        `HP8753E.stop_frequency`, `HP8753E.span_frequency`, `HP8753E.center_frequency`
+        on initialization (float in Hz).
+    :param max_frequency: Optional set the maximum validator for `HP8753E.start_frequency`,
+        `HP8753E.stop_frequency`, `HP8753E.span_frequency`, `HP8753E.center_frequency`
+        on initialization (float in Hz).
+    :param min_power: Optional set the minimum validator for `HP8753E.power` on initialization
+        (float in dBm).
+    :param max_power: Optional set the maximum validator for `HP8753E.power` on initialization
+        (float in dBm).
+
+
+    """
+
+    def __init__(
+        self,
+        adapter=None,
+        name=None,
+        min_frequency=30.0e3,
+        max_frequency=6.0e9,
+        min_power=-70,
+        max_power=10,
+        **kwargs,
+    ):
+        super().__init__(adapter=adapter, name=name, includeSCPI=False, **kwargs)
+
+        self._manu = ""
+        self._model = ""
+        self._fw = ""
+        self._sn = ""
+        self._options = ""
+
+        self.start_frequency_values = [min_frequency, max_frequency]
+        self.stop_frequency_values = [min_frequency, max_frequency]
+        self.center_frequency_values = [min_frequency, max_frequency]
+        self.span_frequency_values = [0, max_frequency - min_frequency]
+
+        self.power_values = [min_power, max_power]
+
+        if name is None:
+            # written this way to pass 'test_all_instruments.py' while allowing the
+            # *IDN? to populate the name of the VNA
+            try:
+                self._manu, self._model, _, self._fw = self.id
+            except ValueError:
+                self._manu = "Hewlett Packard"
+                self._model = "8753E"
+            self._desc = "Vector Network Analyzer"
+            name = self.name = f"{self._manu} {self._model} {self._desc}"
+        else:
+            self.name = name
+
+    ALLOWED_BANDWIDTH = [10, 30, 100, 300, 1000, 3000, 3700, 6000]
+    SCAN_POINT_VALUES = [3, 11, 21, 26, 51, 101, 201, 401, 801, 1601]
+    MEASURING_PARAMETERS = ["S11", "S12", "S21", "S22", "A/R", "B/R", "A/B", "A", "B", "R"]
+    MEASURING_PARAMETER_MAP = {
+        "A/R": "AR",
+        "B/R": "BR",
+        "A/B": "AB",
+        "A": "MEASA",
+        "B": "MEASB",
+        "R": "MEASR",
+    }
+
+    start_frequency = Instrument.control(
+        "STAR?",
+        "STAR %e Hz",
+        """Control the start frequency in Hz. (float).
+        Property is dynamic and allows remapping min/max limits.
```

Comment:
One last comment:
Looking at the dynamic properties, I noticed, that they add `(dynamic)` to the docstring, so you do not have to do it manually.

Reply:
Do you want me to remove the `Property is dynamic and allows remapping min/max limits.` lines from the dynamic properties?

Reply:
Yes, please.

Reply:
Removed `Property is dynamic and allows remapping min/max limits.` Should be go now.